relentless/simulate/lammps.py

```diff
@@ -512,107 +506,87 @@ def to_commands(self, sim):
+                                                                                     Tstart=self.thermostat.T,
+                                                                                     Tstop=self.thermostat.T,
+                                                                                     Tdamp=self.thermostat.tau)]
+        elif self.thermostat is None and isinstance(self.barostat, MTKBarostat):
+            cmds = ['fix {idx} {group_idx} nph iso {Pstart} {Pstop} {Pdamp}'.format(idx=self._fix,
+                                                                                    group_idx='all',
+                                                                                    Pstart=self.barostat.P,
+                                                                                    Pstop=self.barostat.P,
+                                                                                    Pdamp=self.barostat.tau)]
+        elif isinstance(self.thermostat, MTKThermostat) and isinstance(self.barostat, MTKBarostat):
+            cmds = ['fix {idx} {group_idx} npt temp {Tstart} {Tstop} {Tdamp} iso {Pstart} {Pstop} {Pdamp}'.format(idx=self._fix,
+                                                                                                                  group_idx='all',
+                                                                                                                  Tstart=self.thermostat.T,
+                                                                                                                  Tstop=self.thermostat.T,
+                                                                                                                  Tdamp=self.thermostat.tau,
+                                                                                                                  Pstart=self.barostat.P,
+                                                                                                                  Pstop=self.barostat.P,
+                                                                                                                  Pdamp=self.barostat.tau)]
 
         return cmds
 
-class RemoveNVTIntegrator(LAMMPSOperation):
-    """Removes the NVT integrator operation.
-
-    Parameters
-    ----------
-    add_op : :class:`AddNVTIntegrator`
-        The integrator addition operation to be removed.
-
-    Raises
-    ------
-    TypeError
-        If the specified addition operation is not a NVT integrator.
-
-    """
+class RemoveVerletIntegrator(LAMMPSOperation):
     def __init__(self, add_op):
-        if not isinstance(add_op, AddNVTIntegrator):
-            raise TypeError('Addition operation is not AddNVTIntegrator.')
+        if not isinstance(add_op, AddVerletIntegrator):
+            raise TypeError('Addition operation is not AddVerletIntegrator.')
         self.add_op = add_op
 
     def to_commands(self, sim):
         cmds = ['unfix {idx}'.format(idx=self.add_op._fix)]
```

Comment:
To make this easier, maybe you want to make `add_op._extra_fixes` an empty list, then loop over these? That way you don't need to edit this method every time you attach extra fixes.